src/Components/People.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export default class People extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      searchPerson: "",
+      currentPerson: {},
+      currentName: "",
+      error: false,
+    };
+  }
+
+  handleChange = async (event) => {
+    this.setState({
+      searchPerson: event.target.value,
+    });
+  };
+
+  handleSubmit = async (event) => {
+    event.preventDefault();
+
+    const { searchPerson } = this.state;
+    try {
+      const { data } = await axios.get(
+        "https://ghibliapi.herokuapp.com/people",
+      );
+      const personName = data.find((person) => person.name === searchPerson);
+      if (personName === undefined) {
+        this.setState({
+          searchPerson: "",
+          currentPerson: {},
+          currentName: "",
+          error: true,
+        });
+      }
+      this.setState({
+        searchPerson: "",
+        currentPerson: personName,
+        currentName: personName.name,
+        error: false,
+      });
+    } catch (e) {
+      this.setState({
+        searchPerson: "",
+        currentPerson: {},
+        currentName: "",
+        error: true,
+      });
+    }
+  };
+
+  render() {
+    const { searchPerson, currentPerson, currentName, error } = this.state;
+    return (
+      <div>
+        <h1>Search for a Person</h1>
+        <form onSubmit={this.handleSubmit}>
+          <input
+            value={searchPerson}
+            type="text"
+            placeholder="Find Your Person"
+            onChange={this.handleChange}
+          />
+          <button>Submit</button>
+        </form>
+        {error ? (
```

Comment:
I suggest rewriting without using ternary statements. They are meant to be used as a single line expression. Otherwise it can be a little difficult to read. Maybe separate the error message and the person's name